<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_node</class>
 <widget class="QDialog" name="new_node">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start node</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of the node</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>127</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
   <property name="buddy">
    <cstring>ip</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>127</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port number</string>
   </property>
   <property name="buddy">
    <cstring>port</cstring>
   </property>
  </widget>
  <widget class="QSpinBox" name="port">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>66</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1025</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="chck_init">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>267</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Do you want to connect to other node ? </string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ip_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>127</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_port_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="port_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>66</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1025</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_keys">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>124</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Key</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>77</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>new_node</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>new_node</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>ip_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>port_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_ip_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chck_init</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_port_next</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
